LAKMobile/src/screens/SignupScreen.tsx

```diff
@@ -123,7 +122,7 @@ export function SignupScreen({ navigation }: SignupProps) {
           type="phoneNumber"
           isValid={!isSignupPressed || phoneValid}
           errMsg="Valid mobile number required."
-          maxLength={10}
+          maxLength={16}
```

Comment:
Maybe we can leave a comment here stating that the longest phone number with a + is 16 chars? This wasn't super obvious at first glance